<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BubblesSetupWidget</class>
 <widget class="QWidget" name="BubblesSetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Общее</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QCheckBox" name="showLineouts">
        <property name="text">
         <string>Сноски</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="showLabels">
        <property name="text">
         <string>Подписи</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="showDiagramms">
        <property name="text">
         <string>Диаграммы</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>nameEdit_editingFinished()</slot>
  <slot>titleEdit_editingFinished()</slot>
  <slot>scaleEdit_editingFinished()</slot>
  <slot>diagrammTypeChanged(int)</slot>
  <slot>diagrammType_EditTextChanged(QString)</slot>
  <slot>scaleTypeChanged(int)</slot>
  <slot>minDiagrammSize_ValueChanged(double)</slot>
  <slot>maxDiagrammSize_ValueChanged(double)</slot>
  <slot>fluidItemClicked(QListWidgetItem*)</slot>
  <slot>fluidItemRowChanged(int)</slot>
  <slot>fluidBackColorClicked(QColor)</slot>
  <slot>fluidLineColorClicked(QColor)</slot>
  <slot>labelColorChanged(QColor)</slot>
  <slot>labelTemplateEditingFinished()</slot>
  <slot>showLineoutsToggled(bool)</slot>
  <slot>perDayProdToggled(bool)</slot>
  <slot>templateButtonChanged(QString)</slot>
 </slots>
</ui>
